Okta-Application/src/handlers.ts

```diff
@@ -88,10 +91,16 @@ class Resource extends AbstractOktaResource<ResourceModel, Application, Applicat
         });
         // Special case for unusual capitalisation
         result.visibility.hide.ios = (<any>from?.visibility?.hide)?.iOS;
+        delete (<any>result?.visibility?.hide)?.iOS;
         // Delete a couple of unused fields that are returned by the API
         // as they are subject to change server-side
         delete (<any>result)?.lastUpdated;
         delete (<any>result)?.status;
+        delete (<any>result)?._links;
```

Comment:
Instead of doing all the deletes would it be cleaner to do something like this/ #[https://github.com/aws-ia/cloudformation-snowflake-resource-providers/pull/14](Snowflake Database Extra Properties Cleanup)